.github/CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 
 
 # The yoshi-nodejs team is the default owner for nodejs repositories.
-*     @googleapis/yoshi-nodejs
```

Comment:
I believe we've been doing this in the reverse order:

https://github.com/googleapis/nodejs-storage/blob/master/.github/CODEOWNERS#L10

`@googleapis/yoshi-nodejs @googleapis/api-dataproc`

Mind swapping for the sake of consistency (I'm not sure whether it makes a functional difference).

Reply:
Done, thank you for the suggestion.